### ISSUE TITLE
The datepicker calender position bug

### ISSUE THREAD

**Issue by JeskTop** [eternicode/bootstrap-datepicker#410]
Hi.
The top position has some problems.Like this:
![E6490327-CEC9-4EC2-9FD6-E391E6DADB73](https://f.cloud.github.com/assets/1229979/307454/76f745a0-96c2-11e2-845f-8290a30c3ada.png)

```
<div class="control-group string optional">
  <label class="string optional control-label" for="book_publication_date">Publication date</label>
  <div class="controls">
    <input class="string optional input-xlarge" data-behaviour="datepicker" id="book_publication_date" name="book[publication_date]" size="50" type="text">
  </div>
</div>
```
```
$("[data-behaviour~='datepicker']").datepicker({"format": "yyyy-mm-dd", "weekStart": 1, "autoclose": true})
```
The input "publication_date" used 'datepicker' but the position top fix -40px.

**Comment by acrobat** [comment]
This issue should be fixed with the new container option added in #918 and with the extra placement fixes in #1227

**Comment by acrobat** [comment]
Fixed in #1227
